website/docs/r/nrql_alert_condition.html.markdown

```diff
@@ -76,8 +76,8 @@ The following attributes are exported:
 
 ## Import
 
-Alert conditions can be imported using the `id`, e.g.
+Alert conditions can be imported using the a composite ID of `<policy_id>:<condition_id> `id`, e.g.
```

Comment:
looks like you'll want to remove the leftover "the" here.

Reply:
good catch. fixed!